src/pages/vault/tutorials/[...tutorialSlug]/index.tsx

```diff
@@ -20,9 +20,16 @@ export function VaultTutorialPage({
 
 export async function getStaticProps({
   params,
-}): Promise<{ props: TutorialPageProps }> {
+}): Promise<GetStaticPropsResult<TutorialPageProps>> {
   const product = vaultData as LearnProductData
-  return getTutorialPageProps(product, params.tutorialSlug)
+  const data = await getTutorialPageProps(product, params.tutorialSlug)
+
+  // If the tutorial doesn't exist, hit the 404
+  if (!data) {
+    return { notFound: true }
+  }
+
+  return { props: data, revalidate: 300 }
```

Comment:
Not sure of the best `revalidate` number here. Or if that's config we should consolidate.

Reply:
Non-blocking out-of-curiosity question: Before this PR, we weren't returning a `revalidate` value at all, right? 🤔

I am a bit fuzzy on how `revalidate` & ISR works with relation to 404 pages, but if that's the case and we weren't returning `revalidate` previously, I wonder if it'd be possible to handle the 404 case without adding it? (not saying we _should_, just wondering if possible ☺️ )

Reply:
Ya fair, I can make a ticket for this to be something we address on another PR! To chat through how / if we want to add this.